bin/oxfordshire/one-off-remove-inspector-perms

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env perl
+#
+# One off script to remove inspector permissions for all staff.
+
+use strict;
+use warnings;
+use v5.14;
+
+BEGIN {
+    use File::Basename qw(dirname);
+    use File::Spec;
+    my $d = dirname(File::Spec->rel2abs($0));
+    require "$d/../../setenv.pl";
+}
+
+use FixMyStreet::DB;
+
+use Getopt::Long;
+
+my $commit;
+GetOptions(
+    'commit' => \$commit,
+);
+
+if (!$commit) {
+    say "*** DRY RUN ***";
+}
+
+my $body = FixMyStreet::DB->resultset("Body")->find({ name => 'Oxfordshire County Council' });
```

Comment:
Could we make the name, and the list of permissions, command line arguments, then this is a slightly more useful generic tool? And even if time, make it add or delete, so it could also give every staff user at a body a particular permission? Anyway, at least the name/permissions would be good.